.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Bug report
+description: Create a bug report to help us improve
+labels:
+  - bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: OpenWrt version
+      description: The OpenWrt release or commit hash where this bug occurs (use command below).
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ```. /etc/openwrt_release && echo $DISTRIB_REVISION```
```

This division into two fields is "strange", you can add multiline with markdown directly to the descripcion.
```suggestion
  - type: input
    id: version
    attributes:
      label: OpenWrt version
      description: |
        The OpenWrt release or commit hash where this bug occurs. You can use this command:
        ```. /etc/openwrt_release && echo $DISTRIB_REVISION```
    validations:
      required: true
  - type: markdown
    attributes:
      value: |
        
```